Riya Singhal/Golang/arrays.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import "fmt"
+
+func main() {
+	//1-D arrays
+
+	//default values are set to 0
```

Comment:
```suggestion
	// default values are set to 0
```
When you comment, always put space after `//`.

Ditto in all other places